test/app-for-e2e/pkg/database/pets.go

```diff
@@ -0,0 +1,139 @@
+package database
+
+import (
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"strconv"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbattribute"
+	"github.com/awslabs/amazon-ec2-instance-qualifier/ec2-instance-qualifier-app/pkg/crypto"
+)
+
+const tableName = "Pets"
+
+var petCache = make(map[int]Pet)
+
+// Pet is the code representation of a pet entry in the database
+type Pet struct {
+	PetId  int    `json:"PetId"`
```

Comment:
I'm manually assigning ID right now, but that doesn't seem correct (feel like DynamoDB should be doing that). Probably fine for this demo, but still interested in the right way